controls/pcidss_ocp4.yml

```diff
@@ -985,8 +985,13 @@ controls:
   levels:
   - base
   notes: ''
-  status: pending
-  rules: []
+  status: inherently met
```

Comment:
Given that we have several rules implementing this, we might want to switch the status to `automated` here.